### ISSUE TITLE
Need Clarification in I/O Ref about Initialization of Surface Temperatures

### ISSUE THREAD

**Issue by BobHenninger** [NREL/EnergyPlus#4254]
I/O Ref does not state anywhere that surface temperatures are initialized at 23 C.  Searched through the v8.1 I/O Ref and could not find any place where it was stated that surfaces temps were initialized at 23 C.  Helpdesk Ticket 6475

**Comment by mjwitte** [comment]
Here are the related initializations in source code:

```
HeatBalanceManager::InitHeatBalance
            SurfaceWindow.EffInsSurfTemp() = 23.0;

HeatBalanceManager::AllocateHeatBalArrays
        XMAT.dimension( NumOfZones, 23.0 );
        XM2T.dimension( NumOfZones, 23.0 );
        XM3T.dimension( NumOfZones, 23.0 );
        XM4T.dimension( NumOfZones, 23.0 );
        DSXMAT.dimension( NumOfZones, 23.0 );
        DSXM2T.dimension( NumOfZones, 23.0 );
        DSXM3T.dimension( NumOfZones, 23.0 );
        DSXM4T.dimension( NumOfZones, 23.0 );

        ZTAV.dimension( NumOfZones, 23.0 );
        ZTAVComf.dimension( NumOfZones, 23.0 );
        ZT.dimension( NumOfZones, 23.0 );
        TempTstatAir.dimension( NumOfZones, 23.0 );
        MAT.dimension( NumOfZones, 23.0 );
        ZoneTMX.dimension( NumOfZones, 23.0 );
        ZoneTM2.dimension( NumOfZones, 23.0 );

HeatBalanceSurfaceManager::InitThermalAndFluxHistories
        // First do the "bulk" initializations of arrays sized to NumOfZones
        MRT = 23.0; // module level array
        MAT = 23.0; // DataHeatBalFanSys array
        ZT = 23.0;
        ZTAV = 23.0;
        XMAT = 23.0; // DataHeatBalFanSys array
        XM2T = 23.0; // DataHeatBalFanSys array
        XM3T = 23.0; // DataHeatBalFanSys array
        XM4T = 23.0;
        XMPT = 23.0;
        DSXMAT = 23.0; // DataHeatBalFanSys array
        DSXM2T = 23.0; // DataHeatBalFanSys array
        DSXM3T = 23.0; // DataHeatBalFanSys array
        DSXM4T = 23.0;
        ZoneTMX = 23.0; // DataHeatBalFanSys array
        ZoneTM2 = 23.0; // DataHeatBalFanSys array

        ZoneAirHumRatAvg = OutHumRat;
        ZoneAirHumRat = OutHumRat;
        ZoneAirHumRatOld = OutHumRat;

        // "Bulk" initializations of temperature arrays with dimensions (TotSurface,MaxCTFTerms,2)
        TH = 23.0; // module level array
        THM = 23.0; // module level array
        TsrcHist = 23.0;
        TsrcHistM = 23.0;

        SurfaceWindow.FrameTempSurfIn() = 23.0;
        SurfaceWindow.FrameTempSurfInOld() = 23.0;
        SurfaceWindow.FrameTempSurfOut() = 23.0;
        SurfaceWindow.DividerTempSurfIn() = 23.0;
        SurfaceWindow.DividerTempSurfInOld() = 23.0;
        SurfaceWindow.DividerTempSurfOut() = 23.0;

HeatBalanceSurfaceManager::CalcHeatBalanceInsideSurf
        TempInsOld = 23.0;
        RefAirTemp = 23.0;
        TempEffBulkAir = 23.0;

HeatBalFiniteDiffManager::
    Real64 const TempInitValue( 23.0 ); // Initialization value for Temperature

HeatBalFiniteDiffManager::InitHeatBalFiniteDiff
                SurfaceFD( SurfNum ).T = TempInitValue;
                SurfaceFD( SurfNum ).TOld = TempInitValue;
                SurfaceFD( SurfNum ).TT = TempInitValue;

                SurfaceFD( SurfNum ).TD = TempInitValue;
                SurfaceFD( SurfNum ).TDT = TempInitValue;
                SurfaceFD( SurfNum ).TDTLast = TempInitValue;
                SurfaceFD( SurfNum ).TDOld = TempInitValue;
                SurfaceFD( SurfNum ).TDreport = TempInitValue;

HeatBalFiniteDiffManager::InitialInitHeatBalFiniteDiff
            SurfaceFD( Surf ).T = TempInitValue;
            SurfaceFD( Surf ).TOld = TempInitValue;
            SurfaceFD( Surf ).TT = TempInitValue;
            SurfaceFD( Surf ).TD = TempInitValue;
            SurfaceFD( Surf ).TDT = TempInitValue;
            SurfaceFD( Surf ).TDTLast = TempInitValue;
            SurfaceFD( Surf ).TDOld = TempInitValue;
            SurfaceFD( Surf ).TDreport = TempInitValue;

RoomAirModelAirflowNetwork::RAFNData::InitRoomAirModelAirflowNetwork
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTemp = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempX1 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempX2 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempX3 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempX4 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempDSX1 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempDSX2 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempDSX3 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempDSX4 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempT1 = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempTMX = 23.0;
                    RoomAirflowNetworkZoneInfo( LoopZone ).Node( LoopAirNode ).AirTempTM2 = 23.0;

RoomAirModelManager::SharedDVCVUFDataInit
                MATFloor = 23.0;
                XMATFloor = 23.0;
                XM2TFloor = 23.0;
                XM3TFloor = 23.0;
                XM4TFloor = 23.0;
                DSXMATFloor = 23.0;
                DSXM2TFloor = 23.0;
                DSXM3TFloor = 23.0;
                DSXM4TFloor = 23.0;
                MATOC = 23.0;
                XMATOC = 23.0;
                XM2TOC = 23.0;
                XM3TOC = 23.0;
                XM4TOC = 23.0;
                DSXMATOC = 23.0;
                DSXM2TOC = 23.0;
                DSXM3TOC = 23.0;
                DSXM4TOC = 23.0;
                MATMX = 23.0;
                XMATMX = 23.0;
                XM2TMX = 23.0;
                XM3TMX = 23.0;
                XM4TMX = 23.0;
                DSXMATMX = 23.0;
                DSXM2TMX = 23.0;
                DSXM3TMX = 23.0;
                DSXM4TMX = 23.0;
                ZTM1Floor = 23.0;
                ZTM2Floor = 23.0;
                ZTM3Floor = 23.0;
                ZTM1OC = 23.0;
                ZTM2OC = 23.0;
                ZTM3OC = 23.0;
                ZTM1MX = 23.0;
                ZTM2MX = 23.0;
                ZTM3MX = 23.0;
                Zone1Floor = 23.0;
                ZoneMXFloor = 23.0;
                ZoneM2Floor = 23.0;
                Zone1OC = 23.0;
                ZoneMXOC = 23.0;
                ZoneM2OC = 23.0;
                Zone1MX = 23.0;
                ZoneMXMX = 23.0;
                ZoneM2MX = 23.0;
                AIRRATFloor = 0.0;
                AIRRATOC = 0.0;
                AIRRATMX = 0.0;
                ZTOC = 23.0;
                ZTMX = 23.0;
                ZTFloor = 23.0;

                ZTJET = 23.0;
                RoomOutflowTemp = 23.0;
                ZTREC = 23.0;

                MATFloor( ZoneNum ) = 23.0;
                XMATFloor( ZoneNum ) = 23.0;
                XM2TFloor( ZoneNum ) = 23.0;
                XM3TFloor( ZoneNum ) = 23.0;
                XM4TFloor( ZoneNum ) = 23.0;
                DSXMATFloor( ZoneNum ) = 23.0;
                DSXM2TFloor( ZoneNum ) = 23.0;
                DSXM3TFloor( ZoneNum ) = 23.0;
                DSXM4TFloor( ZoneNum ) = 23.0;
                MATOC( ZoneNum ) = 23.0;
                XMATOC( ZoneNum ) = 23.0;
                XM2TOC( ZoneNum ) = 23.0;
                XM3TOC( ZoneNum ) = 23.0;
                XM4TOC( ZoneNum ) = 23.0;
                DSXMATOC( ZoneNum ) = 23.0;
                DSXM2TOC( ZoneNum ) = 23.0;
                DSXM3TOC( ZoneNum ) = 23.0;
                DSXM4TOC( ZoneNum ) = 23.0;
                MATMX( ZoneNum ) = 23.0;
                XMATMX( ZoneNum ) = 23.0;
                XM2TMX( ZoneNum ) = 23.0;
                XM3TMX( ZoneNum ) = 23.0;
                XM4TMX( ZoneNum ) = 23.0;
                DSXMATMX( ZoneNum ) = 23.0;
                DSXM2TMX( ZoneNum ) = 23.0;
                DSXM3TMX( ZoneNum ) = 23.0;
                DSXM4TMX( ZoneNum ) = 23.0;
                ZTM1Floor( ZoneNum ) = 23.0;
                ZTM2Floor( ZoneNum ) = 23.0;
                ZTM3Floor( ZoneNum ) = 23.0;
                Zone1Floor( ZoneNum ) = 23.0;
                ZoneMXFloor( ZoneNum ) = 23.0;
                ZoneM2Floor( ZoneNum ) = 23.0;
                ZTM1OC( ZoneNum ) = 23.0;
                ZTM2OC( ZoneNum ) = 23.0;
                ZTM3OC( ZoneNum ) = 23.0;
                Zone1OC( ZoneNum ) = 23.0;
                ZoneMXOC( ZoneNum ) = 23.0;
                ZoneM2OC( ZoneNum ) = 23.0;
                ZTM1MX( ZoneNum ) = 23.0;
                ZTM2MX( ZoneNum ) = 23.0;
                ZTM3MX( ZoneNum ) = 23.0;
                Zone1MX( ZoneNum ) = 23.0;
                ZoneMXMX( ZoneNum ) = 23.0;
                ZoneM2MX( ZoneNum ) = 23.0;

                ZTOC( ZoneNum ) = 23.0;
                ZTMX( ZoneNum ) = 23.0;
                ZTFloor( ZoneNum ) = 23.0;

                ZTJET( ZoneNum ) = 23.0;

                Tin( ZoneNum ) = 23.0;

RoomAirModelUserTempPattern::InitTempDistModel
            AirPatternZoneInfo( ZoneNum ).TairMean = 23.0;
            AirPatternZoneInfo( ZoneNum ).Tstat = 23.0;
            AirPatternZoneInfo( ZoneNum ).Tleaving = 23.0;
            AirPatternZoneInfo( ZoneNum ).Texhaust = 23.0;
            AirPatternZoneInfo( ZoneNum ).Gradient = 0.0;
            for ( SurfNum = 1; SurfNum <= AirPatternZoneInfo( ZoneNum ).totNumSurfs; ++SurfNum ) {
                AirPatternZoneInfo( ZoneNum ).Surf( SurfNum ).TadjacentAir = 23.0;
            }

```

**Comment by mjwitte** [comment]
Closed via #5352
